tests/unit/test-class-wc-payment-gateway-wcpay-process-payment.php

```diff
@@ -218,11 +218,6 @@ public function test_intent_status_success() {
 		// Arrange: Return a successful response from create_and_confirm_intention().
 		$intent = WC_Helper_Intention::create_intention();
 
-		$this->mock_customer_service
-			->expects( $this->once() )
-			->method( 'get_customer_id_by_user_id' )
-			->willReturn( 'cus_mock' );
-
```

Comment:
This expectation was a duplicate of one just a few lines above, it's actually ignored by PHPUnit.